src/jira/client/jira-client.ts

```diff
@@ -421,6 +454,27 @@ const updateIssueKeysFor = (resources, func) => {
 	return resources;
 };
 
+/**
+ * Runs the mutatingFunc on the association values field for each entity resource
+ * Assumption is that the transformed resource only has one association which is for
+ * "issueIdOrKeys" association.
+ */
+const updateIssueKeyAssociationValuesFor = (resources: JiraRemoteLink[], mutatingFunc: any): JiraRemoteLink[] => {
```

Comment:
shouldn't the func be of type `mutatingFunc:(value:string[]) =>  string[]`?

Reply:
Nope because mutatingFunc can be `dedup` or `truncate` and they both accept any type of array, nothing guarantees those will always be `string[]`

Reply:
both dedup needs to be a `string[]` input, truncate can be `unknown[]`, but in your case, for this function, it needs to be able to accept a `string[]`